src/nbl/ext/MitsubaLoader/CMitsubaLoader.cpp

```diff
@@ -378,6 +378,7 @@ static core::smart_refctd_ptr<asset::ICPURenderpassIndependentPipeline> createPi
 
 	SRasterizationParams rasterParams;
 	rasterParams.faceCullingMode = asset::EFCM_NONE;
+	rasterParams.frontFaceIsCCW = 1;
```

Comment:
and what was it before?

Reply:
1 is default, just wanted to make it explicit